feat: login, signup pages

ddivyansh18 opened the pull request:
- Added login, signup pages to NyceKart
![image](https://user-images.githubusercontent.com/41231917/153766448-3e2baa4e-7c98-462b-b2c1-8ade1ad464f8.png)

ddivyansh18 committed:
feat: login, signup pages

- Added login, signup pages to NyceKart

ddivyansh18 committed:
chore: removed commented code

- Removed commented and unused CSS property

manish591 commented on auth/login.html:
```diff
@@ -0,0 +1,109 @@
+    </nav>
+
+    <div class="page-content-container">
+      <section class="login-container card-shadow column-container white-bg">
```
Use form element for login container. It is semantically right and gives you more control over sections.

ddivyansh18 replied:
Added form element for both the pages

manish591 commented on auth/login.html:
```diff
@@ -0,0 +1,109 @@
+        </div>
+        <div class="login-field-container">
+          <label for="password">Password</label>
+          <input type="password" name="" id="password" />
```
Add a required attribute here.

ddivyansh18 replied:
@manish591 Fixed

ddivyansh18 committed:
fix: more semantic HTML for forms

- Replaced section tag with more semantically correct form tag
- Removed button and added an input with type="submit" which is appropriate for a form

ddivyansh18 committed:
fix: missing tag value, required fields

- Added the missing value in name property of password input
- Made the email and password fields mandatory for submission of form
